Zend/zend_operators.h

```diff
@@ -919,6 +920,70 @@ static zend_always_inline void zend_unwrap_reference(zval *op) /* {{{ */
 }
 /* }}} */
 
+static zend_always_inline const char *strnichr(const char *ptr, char value, size_t num)
+{
+	char value_lower = zend_tolower_ascii(value);
+	char value_upper = zend_toupper_ascii(value);
+
+	if (value_lower == value_upper) {
+		return memchr(ptr, value, num);
```

Comment:
`memchr()` returns a `void*`, but the function is supposed to return a `const char*`, so you need to cast manually.

Reply:
Thanks, will fix that. I didn't get any errors locally for some reason (or maybe hidden behind a wall of text).